### PR TITLE
Add Pierre Tessier as an approver

### PR DESCRIPTION
**Pull request by dmitryax**
@puckpuck is a maintainer of https://github.com/open-telemetry/opentelemetry-demo and an active contributor to the demo chart.

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]
Welcome aboard @puckpuck! Please approve this PR if you accept the role.

**Comment by puckpuck** [comment]
Approved!

**Review by puckpuck — APPROVED** [review]

**Comment by TylerHelmuth** [comment]
@puckpuck be on the lookout for an invite to join the approvers team.  @dmitryax you're the only team maintainer so you'll need to do the invite.

**Comment by dmitryax** [comment]
> @dmitryax you're the only github team maintainer so you'll need to do the invite.

done 👍

**Pull request merged**
